code/auth-service/src/__mocks__/winston.js

```diff
@@ -44,4 +44,6 @@ function debug(message, data, metadata) {
   messages.debug.push({ message, data, metadata });
 }
 
-export default { info, warn, error, debug, clearMessages, getInfoMessages, getWarnMessages, getErrorMessages, getDebugMessages };
+export default {
```

Comment:
Presumably you changed this to appease linting? I think we have added new linting config in other projects to make this valid. I think it is
```
"object-curly-newline": ["error", { "consistent": true }],
```
 in `.eslintrc`